docs/en/patterns/contextual-menu.md

```diff
@@ -5,27 +5,34 @@ title: Contextual menu
 
 ## Contextual menu
 
-A contextual menu can be applied to any button, link or navigation item that requires a secondary menu. To interact with the menu it will require some javascript to hide/show each pattern. This achieved by finding the toggle element `p-contextual-menu__toggle` and what it controls `aria-controls`. The target element will be hidden or shown with `aria-hidden="true"` or `false`. The control element will also change `aria-expanded` so and screen readers will know it's active.
+<hr>
```

Comment:
You need a blank line after the HR in order to make the following text a paragraph (it's a markdown quirk)

![image](https://user-images.githubusercontent.com/118614/54023063-e46b2f00-418b-11e9-9160-eb60b1ca5bba.png)


```suggestion
<hr>

```